templates/inspec/plural_resource.erb

```diff
@@ -29,7 +29,7 @@ name = "google_#{product_ns.downcase}_#{object.name.underscore}"
   filter_table_config = FilterTable.create
 <% object.all_user_properties.reject(&:exclude_plural).each do |prop| -%>
 
-  <%= "filter_table_config.add(:#{prop.out_name.pluralize}, field: :#{prop.out_name})" -%>
+  <%= "filter_table_config.add(:#{(prop.override_name || prop.out_name).pluralize}, field: :#{prop.override_name || prop.out_name})" -%>
```

Comment:
This will replace the current api name with the overridden one. In this case we would need to add docs changes as well, telling the user to use the overridden name rather than the api name. It wouldn't be a drastic change, similar to what is here.


I would prefer to just add another filter table config for the overridden name that points to the api name.
For example, regions would still parse the `name` value from the API response into their `name` array field, but would expose that `name` array as both `names` and `region_names`.

Reply:
The docs should already be consistent but I'm happy to do that if preferred.  Thought this route would be less confusing to avoid repetition e.g. 
```
  * `region_ids`: an array of `google_compute_region` id
  * `region_names`: an array of `google_compute_region` name
  * `ids`: an array of `google_compute_region` id
  * `names`: an array of `google_compute_region` name
```

Reply:
Ah right, missed the change in the doc template